Ross-Ulbricht-Miami-Bitcoin-Conference/It/ch001-Ross-Ulbricht-Miami-Bitcoin-Conference.md

```diff
@@ -0,0 +1,426 @@
+1. 
+ Ciao, Ross.  
+  Sono Peter della rivista Bitcoin Magazine. Sei pronto a partire?  
+  Si. Bene, allora...  
+  Salve, sono Ross Ulrich e vi chiamo oggi  
+  da un penitenziario federale di massima sicurezza.  
+  Non abbiamo avuto molto tempo per stare insieme oggi, e non so  
+  se avrò un'altra occasione di parlarle in questo modo.  
+  Dico tutto quello che posso, ma quando è ora di andare,  
+  dovrò riattaccare e tornare in cella.  
+  Ho perso la mia libertà.  
+  È di questo che voglio parlarle oggi.  
+  Voglio che capiate cosa significa perdere la libertà.  
+  Ma prima parliamo di bitcoin.  
+  Ero presente durante i primi giorni di bitcoin.  
+  All'epoca, bitcoin mi ha fatto sentire come se tutto fosse possibile.  
+ Bitcoin era aperto a tutti, giusto? 
+  
+ 2. 
+ Mi piaceva molto,  
+  come se avesse rivelato il campo di gioco.  
+  Quando l'idea di bitcoin è scattata davvero dentro di me, mi sono entusiasmato.  
+  Ho pensato: con bitcoin posso provare a fare qualcosa che faccia davvero la differenza.  
+  E comunque, prima di essere messo in  
+  in prigione, non c'erano tutte queste  
+  criptovalute, i token e tutto il resto.  
+  Mi sono perso tutto questo.  
+  Quindi per me è tutto un'unica cosa.  
+  I fork, le nuove blockchain, tutto quanto.  
+  Quindi, quando dico bitcoin,  
+  non faccio queste distinzioni.  
+  Per me può sembrare un po' banale,  
+   ma siamo tutti una grande famiglia.  
+  Allora ero entusiasta,  
+  ma ero anche molto impaziente.  
+  Ho visto cosa poteva fare bitcoin per la libertà e l'uguaglianza,  
+  ma non mi sono preso il tempo per capirlo davvero. 
+  
+ 3. 
+ Non ne ho apprezzato appieno i principi.  
+  Si basa su cose come l'immutabilità e il consenso e, naturalmente, la decentralizzazione.  
+  Avevo tanti grandi sogni per bitcoin e la cosa più bella è che pian piano quei sogni si stanno realizzando.  
+  Questo grazie a voi.  
+  State trasformando quei sogni in realtà.  
+  State facendo quello che io non ho avuto la pazienza di fare.  
```

Comment:
Correggere `la pazienza di fare` con `la pazienza di fare per questi ultimi otto anni in prigione.`